mocca-resiliency4j/src/main/java/com/paypal/mocca/client/MoccaResilience4j.java

```diff
@@ -126,5 +129,43 @@ public Builder bulkhead(final Bulkhead bulkhead) {
             this.feignBuilder = feignBuilder.withBulkhead(bulkhead);
             return this;
```

Comment:
Please, add `Fallback` on line 44 above.
Also, please don't forget to change end user documentation markdown file.

Thanks.